src/namespace.c

```diff
@@ -152,7 +152,7 @@ cc_oci_str_to_ns (const char *str)
  */
 gboolean cc_oci_ns_join(struct oci_cfg_namespace *ns)
 {
-	int fd = -1;
+	int fd;
```

Comment:
any special reason to not initialize this fd, it is reassigned in the test lines, but if the code changes in the future would be better to initialize  it